add-on/panels/js/roomViews.jsx

```diff
@@ -342,7 +342,7 @@ loop.roomViews = (function(mozL10n) {
                 localPosterUrl={this.props.localPosterUrl}
                 localSrcMediaElement={this.state.localSrcMediaElement}
                 localVideoMuted={this.state.videoMuted}
-                matchMedia={this.state.matchMedia || window.matchMedia.bind(window)}
+                matchMedia={this.state.matchMedia || window.matchMedia}
```

Comment:
So now you've done the nice changes, I'm not sure that matchMedia is actually used any more - looks like we dropped it at some stage. Maybe just drop it all now?